pkg/specgen/specgen.go

```diff
@@ -328,6 +328,9 @@ type ContainerCgroupConfig struct {
 // ContainerNetworkConfig contains information on a container's network
 // configuration.
 type ContainerNetworkConfig struct {
+	// Aliases are a list of network-scoped aliases for container
+	// Optional
+	Aliases []string `json:"aliases"`
```

Comment:
This should be map[string]string to properly support aliases via HTTP API (that allows them to be set per-network)